### PR TITLE
istio-security-004 content

### PR DESCRIPTION
**Pull request by pmerrison**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/news/security/istio-security-2023-004/index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+skip_seealso: true
+---
+
+{{< security_bulletin >}}
+
+## CVE
+
+### Envoy CVEs
+
+- __[CVE-2023-48889](https://nvd.nist.gov/vuln/detail/CVE-2023-44487)__: (CVSS Score 7.5, High): HTTP/2 denial of service
```
Comment (ericvn) [inline content/en/news/security/istio-security-2023-004/index.md:20]:
```suggestion
- __[CVE-2023-48887](https://nvd.nist.gov/vuln/detail/CVE-2023-44487)__: (CVSS Score 7.5, High): HTTP/2 denial of service
```

File: content/en/news/releases/1.17.x/announcing-1.17.7/index.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+This release fixes the security vulnerabilities described in our Oct 10th post, [ISTIO-SECURITY-2023-004](/news/security/istio-security-2023-004).
+
+This release note describes what’s different between Istio 1.17.6 and 1.17.7.
+
+{{< relnote >}}
+
+## Security update
+
+- __[CVE-2023-48889](https://nvd.nist.gov/vuln/detail/CVE-2023-44487)__: (CVSS Score 7.5, High): HTTP/2 denial of service
```
Comment (ericvn) [inline content/en/news/releases/1.17.x/announcing-1.17.7/index.md:18]:
```suggestion
- __[CVE-2023-48887](https://nvd.nist.gov/vuln/detail/CVE-2023-44487)__: (CVSS Score 7.5, High): HTTP/2 denial of service
```

File: content/en/news/releases/1.19.x/announcing-1.19.2/index.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+This release fixes the security vulnerabilities described in our Oct 10th post, [ISTIO-SECURITY-2023-004](/news/security/istio-security-2023-004).
+
+This release note describes what’s different between Istio 1.19.1 and 1.19.2.
+
+{{< relnote >}}
+
+## Security update
+
+- __[CVE-2023-48889](https://nvd.nist.gov/vuln/detail/CVE-2023-44487)__: (CVSS Score 7.5, High): HTTP/2 denial of service
```
Comment (ericvn) [inline content/en/news/releases/1.19.x/announcing-1.19.2/index.md:18]:
```suggestion
- __[CVE-2023-48887](https://nvd.nist.gov/vuln/detail/CVE-2023-44487)__: (CVSS Score 7.5, High): HTTP/2 denial of service
```

**Comment by pmerrison** [comment]
> You'll also want to update `docs/releases/supported-releases/docs/releases/supported-releases/#supported-releases-without-known-common-vulnerabilities-and-exposures-cves`

Thanks, done

File: content/en/news/security/istio-security-2023-004/index.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+{{< security_bulletin >}}
+
+## CVE
+
+### Envoy CVEs
+
+- __[CVE-2023-48887](https://nvd.nist.gov/vuln/detail/CVE-2023-44487)__: (CVSS Score 7.5, High): HTTP/2 denial of service
+
+
```
Comment (ericvn) [inline content/en/news/security/istio-security-2023-004/index.md:22]:
```suggestion

```

Comment (ericvn) [reply content/en/news/security/istio-security-2023-004/index.md:22]: Fixes a lint error

**Comment by ericvn** [comment]
Labeling DNM so the RMs can merge when needed.

**Review by ericvn — APPROVED** [review]

**Comment by ericvn** [comment]
/test doc.test.profile-default

Flake

File: content/en/news/security/istio-security-2023-004/index.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+skip_seealso: true
+---
+
+{{< security_bulletin >}}
+
+## CVE
+
+### Envoy CVEs
+
+- __[`CVE-2023-48887`](https://nvd.nist.gov/vuln/detail/CVE-2023-44487)__: (CVSS Score 7.5, High): HTTP/2 denial of service
```
Comment (dhawton) [inline content/en/news/security/istio-security-2023-004/index.md:20]:
```suggestion
- __[`CVE-2023-44487`](https://nvd.nist.gov/vuln/detail/CVE-2023-44487)__: (CVSS Score 7.5, High): HTTP/2 denial of service
```

File: content/en/news/releases/1.19.x/announcing-1.19.2/index.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+This release fixes the security vulnerabilities described in our Oct 10th post, [`ISTIO-SECURITY-2023-004`](/news/security/istio-security-2023-004).
+
+This release note describes what’s different between Istio 1.19.1 and 1.19.2.
+
+{{< relnote >}}
+
+## Security update
+
+- __[`CVE-2023-48887`](https://nvd.nist.gov/vuln/detail/CVE-2023-44487)__: (CVSS Score 7.5, High): HTTP/2 denial of service
```
Comment (dhawton) [inline content/en/news/releases/1.19.x/announcing-1.19.2/index.md:18]:
```suggestion
- __[`CVE-2023-44487`](https://nvd.nist.gov/vuln/detail/CVE-2023-44487)__: (CVSS Score 7.5, High): HTTP/2 denial of service
```

File: content/en/news/releases/1.18.x/announcing-1.18.4/index.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+This release fixes the security vulnerabilities described in our Oct 10th post, [`ISTIO-SECURITY-2023-004`](/news/security/istio-security-2023-004).
+
+This release note describes what’s different between Istio 1.18.3 and 1.18.4.
+
+{{< relnote >}}
+
+## Security update
+
+- __[`CVE-2023-48887`](https://nvd.nist.gov/vuln/detail/CVE-2023-44487)__: (CVSS Score 7.5, High): HTTP/2 denial of service
```
Comment (dhawton) [inline content/en/news/releases/1.18.x/announcing-1.18.4/index.md:18]:
```suggestion
- __[`CVE-2023-44487`](https://nvd.nist.gov/vuln/detail/CVE-2023-44487)__: (CVSS Score 7.5, High): HTTP/2 denial of service
```

File: content/en/news/releases/1.17.x/announcing-1.17.7/index.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+
+This release fixes the security vulnerabilities described in our Oct 10th post, [`ISTIO-SECURITY-2023-004`](/news/security/istio-security-2023-004).
+
+This release note describes what’s different between Istio 1.17.6 and 1.17.7.
+
+{{< relnote >}}
+
+## Security update
+
+- __[`CVE-2023-48887`](https://nvd.nist.gov/vuln/detail/CVE-2023-44487)__: (CVSS Score 7.5, High): HTTP/2 denial of service
```
Comment (dhawton) [inline content/en/news/releases/1.17.x/announcing-1.17.7/index.md:18]:
```suggestion
- __[`CVE-2023-44487`](https://nvd.nist.gov/vuln/detail/CVE-2023-44487)__: (CVSS Score 7.5, High): HTTP/2 denial of service
```

**Review by dhawton — CHANGES_REQUESTED** [review]
CVE link text differed from the CVE number

File: content/en/news/security/istio-security-2023-004/index.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
+---
+title: ISTIO-SECURITY-2023-004
+subtitle: Security Bulletin
+description: CVE reported by Envoy.
+cves: [CVE-2023-48887]
```
Comment (dhawton) [inline content/en/news/security/istio-security-2023-004/index.md:5]:
```suggestion
cves: [CVE-2023-44487]
```

Comment (dhawton) [reply content/en/news/security/istio-security-2023-004/index.md:5]: The CVE is linked to 44487 but is texted as 48887. I believe 44487 is correct but we're mixing between the two

File: content/en/news/releases/1.17.x/announcing-1.17.7/index.md (hunk 8 above)
Comment (pmerrison) [reply content/en/news/releases/1.17.x/announcing-1.17.7/index.md]: Thanks Daniel, I've resolved all of these

File: content/en/docs/releases/supported-releases/index.md [hunk 10]
```diff
@@ -70,9 +70,9 @@ Please keep up-to-date and use a supported version.
 
 | Minor Releases   | Patched versions with no known CVEs                  |
 | ---------------- | ---------------------------------------------------- |
-| 1.19.x           | 1.19.0+                                              |
-| 1.18.x           | 1.18.2+                                              |
-| 1.17.x           | 1.17.5+                                              |
+| 1.19.x           | 1.19.2+                                              |
+| 1.18.x           | 1.18.4+                                              |
```
Comment (keithmattix) [inline content/en/docs/releases/supported-releases/index.md:74]: We ended up having to use 1.18.5 because the charts had already been published for 1.18.4

Comment (pmerrison) [reply content/en/docs/releases/supported-releases/index.md]: I forgot to update that table, I've put in the correct versions (1.17.8, 1.18.5, 1.19.3)

**Review by keithmattix — APPROVED** [review]

File: content/en/news/security/istio-security-2023-004/index.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
+---
+title: ISTIO-SECURITY-2023-004
+subtitle: Security Bulletin
+description: CVE reported by Envoy.
```
Comment (ericvn) [inline content/en/news/security/istio-security-2023-004/index.md:4]:
```suggestion
description: CVEs reported by Envoy and Go.
```

**Review by dhawton — APPROVED** [review]

**Pull request merged**